docs/reference/dep-005.md

```diff
@@ -36,7 +36,7 @@ $ cp -a myplugin/ $(draft home)/plugins/
 ```
 
 ## Building Plugins
-
+// helm chart
```

Comment:
Did you mean to leave `//helm chart` in here? The formatting looks somewhat confusing to me.

Reply:
Sorry about that, I had meant to remove that. Doing so now.